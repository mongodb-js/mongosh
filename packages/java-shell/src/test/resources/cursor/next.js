// before
db.coll.insert({name: "value1", v: 1});
db.coll.insert({name: "value2", v: 2});
db.coll.insert({name: "value2", v: 3});
// command
db.coll.find().next();
// clear
db.coll.drop();