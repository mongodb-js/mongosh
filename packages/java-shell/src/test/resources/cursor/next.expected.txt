{ "_id": <ObjectID>, "name": "value1", "v": 1 }